### PR TITLE
Add page: Tips for contributors and reviewing

### PR DESCRIPTION
**Pull request by sypets**
Related: #58
Resolves: #113
Related: #119

This mostly adds information from the Wiki page: https://wiki.typo3.org/Gerrit_Review_Workflow

### CODE REVIEW

File: Documentation/HandlingAPatch/Review.rst [hunk 1]
```diff
@@ -145,6 +147,43 @@ As soon as the patch has reached the approved status by getting a +2 on
...
+  what we should do here. I think we should have a statement by x or y who have
+  a deeper knowledge of this subsystem to have an eye on that. I do not want
+  this patch to be merged until this was sorted out and will vote -1 for now
+  for this reason."
+
+Hints on voting -2
+------------------
+
+A -2 vote by an active contributor blocks a patch from merging. In contrast to
+-1, a -2 persists new patch sets is a "veto". Use with care. With a -2, you
```
Comment (susannemoog) [inline Documentation/HandlingAPatch/Review.rst:179]: "a -2 persists new patch sets is a "veto"" ... sounds weird.

File: Documentation/HandlingAPatch/Review.rst [hunk 2]
```diff
@@ -175,3 +214,29 @@ Consider these rules when comparing patches:
 * If the patch needed to be rebased onto current master, the changeset might
   contain the changes due to rebasing. So better check the diff between base
   and most recent version in this case.
+
+How to handle [WIP] patches
```
Comment (susannemoog) [inline Documentation/HandlingAPatch/Review.rst:218]: section is generally ok, should however mention the gerrit WIP feature (as that is now integrated into gerrit directly and does not need to be added to the commit message)

File: Documentation/HandlingAPatch/Review.rst [hunk 3]
```diff
@@ -175,3 +214,29 @@ Consider these rules when comparing patches:
...
+
+Better: "Hey, I worked in this area and came up with this WIP patch for now. I
+wanted to show into which direction this patch is leading, but we currently have
+some open questions. However, it would be great if you can give me feedback to
+the general approach at this early state already to decide if it is worth
+following this solution to its end."
+
+Having too many WIP patches in the review queue is not really helpful. Consider
+to fork the project in github or somewhere else and push to gerrit again if
+you patches evolved.
```
Comment (susannemoog) [inline Documentation/HandlingAPatch/Review.rst:242]: you -> your

**Review by susannemoog — CHANGES_REQUESTED** [review]

File: Documentation/HandlingAPatch/Review.rst (hunk 2 above)
Comment (sypets) [reply Documentation/HandlingAPatch/Review.rst:218]: I am not sure I know 100% how it should work now. I changed the sentence and now link to the cheat sheet which shows the command `git push origin HEAD:refs/for/master%wip`.

**Review by susannemoog — APPROVED** [review]

**Pull request merged**